pkgs/development/python-modules/beancount/default.nix

```diff
@@ -42,7 +54,6 @@ buildPythonPackage rec {
         generate a variety of reports from them, and provides a web interface.
     '';
     license = licenses.gpl2;
```

Comment:
Could you please update the license in the process?  It's [`gpl2Only`](https://github.com/beancount/beancount#copyright-and-license) as `gpl2` is obsolete.

Thanks

Reply:
done.